examples/basics/ontologies.ipynb

```diff
@@ -46,7 +46,41 @@
         "!pip install labelbox"
```

Comment:
<p>Is it possible to include in the this PR the changes we introduce in 3.40.0 https://docs.labelbox.com/changelog/4-3-23 </p><p>It is just a few lines, you could copy/paste from this branch - : https://github.com/Labelbox/labelbox-python/blob/ao_ontology_nb/examples/basics/ontologies.ipynb </p><p>This is out of scope but it would be great if we can include it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1063/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>